allennlp/models/encoder_decoders/simple_seq2seq.py

```diff
@@ -280,8 +280,6 @@ def decode(self, output_dict: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor
             predicted_tokens = [self.vocab.get_token_from_index(x, namespace="target_tokens")
                                 for x in indices]
             all_predicted_tokens.append(predicted_tokens)
-        if len(all_predicted_tokens) == 1:
```

Comment:
This is removing the batch dimension, which we don't do here anymore?  Is that the issue?

Reply:
Yep, exactly. Doing this results in the decoded output being the first word if there is a single element in the batch.

Reply:
Exactly, I also fixed this issue in my local version.